Resources/Textures/Clothing/Head/Helmets/paramedhelm.rsi/meta.json

```diff
@@ -1,26 +1,26 @@
 {
-  "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from paradise station at commit https://github.com/ParadiseSS13/Paradise/commit/e5e584804b4b0b373a6a69d23afb73fd3c094365",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by DSC@Cabbage#9633 (561159087765848084)",

```

Comment:
```suggestion
    "copyright": "Made by DSC@cababge#9633 (561159087765848084)",

```
:p

Reply:
:godmode:

Reply:
oh shoot you're right

Reply:
aw man now I gotta go and change it across all my prs

Reply:
just make a single PR with a funny file search find and replace after they're all merged